api/everycache_api/api/views.py

```diff
@@ -16,7 +16,8 @@
 api = Api(blueprint)
 
 api.add_resource(UserList, "/users", endpoint="users_list")
-api.add_resource(UserResource, "/users/<string:username>", endpoint="user_by_username")
```

Comment:
This change is not needed. Please ensure your formatter's line length is set to 88